### DISCUSSION TITLE
图标

### DISCUSSION THREAD

**Discussion by jenboy** [hcjike/halo-theme-dream2.0-plus#19] [General]
你好，我想问一下这种图标是在哪找  

ri-rss-fill  类似这种的

![image](https://github.com/zsjy/halo-theme-dream2.0-plus/assets/121419174/69c9b53e-a54f-4b19-aa27-c5ca487e11b1)

**Comment by mjsoftking** [comment]
https://www.remixicon.cn/

这个讨论居然没有邮件通知
